src/extension/atlas/ExtendedArrayMesh.cpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "atlas_unwrap.hpp"
+#include "ExtendedArrayMesh.hpp"
+
+#include <godot_cpp/core/class_db.hpp>
+
+#include <godot_cpp/classes/array_mesh.hpp>
+
+namespace Thrive
+{
+	void ExtendedArrayMesh::_bind_methods()
```

Comment:
You should run clang-format on all the cpp and hpp files added / changed by this PR.